src/lib/accessibility/FocusZone/FocusZone.tsx

```diff
@@ -747,9 +747,20 @@ export class FocusZone extends React.Component<FocusZoneProps> implements IFocus
         this.props.isRtl,
         (activeRect: ClientRect, targetRect: ClientRect) => {
           let distance = -1
+          let topBottomComparison
```

Comment:
This is not quite a descriptive name.
(don't ask me for proposal for better one)